spec/components/cooked_post_processor_spec.rb

```diff
@@ -291,7 +291,7 @@
 
         # Fake a loading image
         optimized_image = OptimizedImage.create!(
-          url: 'http://a.b.c/10x10.png',
+          url: '/uploads/default/10x10.png',
```

Comment:
Since we're changing this in the PR, I think it is a good time to fix it such that the URL actually matches the URL format for an optimized image that the app creates. I've been bitten many time by bugs with uploads where we mock URLs with random format in the tests.